opal/mca/common/ofi/Makefile.am

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+#                         University Research and Technology
+#                         Corporation.  All rights reserved.
+# Copyright (c) 2004-2013 The University of Tennessee and The University
+#                         of Tennessee Research Foundation.  All rights
+#                         reserved.
+# Copyright (c) 2004-2009 High Performance Computing Center Stuttgart,
+#                         University of Stuttgart.  All rights reserved.
+# Copyright (c) 2004-2005 The Regents of the University of California.
+#                         All rights reserved.
+# Copyright (c) 2011-2013 NVIDIA Corporation.  All rights reserved.
+# Copyright (c) 2014      Cisco Systems, Inc.  All rights reserved.
+# Copyright (c) 2015      Intel, Inc. All rights reserved.
+# Copyright (c) 2017      Los Alamos National Security, LLC.  All rights
+#                         reserved.
+# Copyright (c) 2019      Hewlett Packard Enterprise. All rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+# A word of explanation...
+#
+# This library is linked against various MCA components because the
+# support for ofis is needed in various places.
+#
+# Note that building this common component statically and linking
+# against other dynamic components is *not* supported!
+
+AM_CPPFLAGS = $(opal_common_ofi_CPPFLAGS) -I$(prefix)/include
```

Comment:
The `-I$(prefix)/include` is weird here -- can you explain what it is for?

Specifically: we should not be relying on anything in the installation tree while building (because the installation tree does not exist yet).

Reply:
No there is no good reason for it. I have included that when I wasn't understanding the build process and it fixed my problem at the time. But now I see that the problem was actually that opal_common_ofi_CPPFLAGS was renamed to opal_ofi_CPPFLAGS (as off f5e1a672ccd5db127e85e1e8f6bcfeb8a8b04527) and I missed to fix that line...